### PR TITLE
Do not log stack trace when event type does not exist

### PR DESCRIPTION
**Pull request by lmontrieux**
# Do not log stack trace when event type does not exist

> Zalando ticket : ARUHA-1581
## Description
Do not log stack trace when event type does not exist

## Review
- [ ] Tests
- [ ] Documentation
- [ ] CHANGELOG

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,11 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
 
+## [2.6.3] = 2018-04-10
```
Comment (v-stepanov) [inline CHANGELOG.md]: `-` instead of `=`

Comment (lmontrieux) [reply CHANGELOG.md]: 1aaddd6

**Comment by v-stepanov** [comment]
👍

**Comment by lmontrieux** [comment]
Deploy validation please

**Comment by lmontrieux** [comment]
👍

**Pull request merged**